### ISSUE TITLE
link create failure: cannot unmarshal DNS message

### ISSUE THREAD

**Issue by ajssmith** [skupperproject/skupper#546]
With 0.7 release, when attempting to link two ibm cloud OenShift cluster instances, I get the following error:

`Link link1 not active (Failed to redeem claim: Post https://claims-skupper.ansmith-vpc2-cluster-153f1de160110098c1928a6c05e19444-0000.us-south.containers.appdomain.cloud:443/b9e422d6-f632-11eb-a7bd-8c1645a1bc1c: dial tcp: lookup claims-skupper.ansmith-vpc2-cluster-153f1de160110098c1928a6c05e19444-0000.us-south.containers.appdomain.cloud on 172.21.0.10:53: cannot unmarshal DNS message)`

I can successfully attach from minikube on my laptop using the same token.

Looking around, there have been issues reported that appear related to go resolver. Need to research further.

**Comment by grs** [comment]
What happens if you try to curl that url in the place you see the error?

(It looks like it is an issue outside of skupper to me).

**Comment by ajssmith** [comment]
Yes, I tried to curl the url and it was succesful (sorry I left that out). Yes, from searching it seems to be an issue external to skupper.

**Comment by grs** [comment]
You probably saw this already: https://bugzilla.redhat.com/show_bug.cgi?id=1970889 ?
